tests/yam/agama/add_home_file_system.pm

```diff
@@ -0,0 +1,33 @@
+## Copyright 2023 SUSE LLC
```

Comment:
could you check if we could use one of the unified files from #17629 
In theory we should just need a single file to trigger this.

Reply:
Do you mean to use the file `tests/yam/agama/agama.pm` by set the _var_ `AGAMA_TEST` to trigger this?

Reply:
yes, that PR still needs more improvements, but if you could try to use it and even give feedback there would help us. But I'm aware it is a bit out of scope, we could also merge this and have more unification tasks (but I prefer less tickets and more PRs :) ) up to you.